app/templates/app/document_detail.html

```diff
@@ -8,7 +8,12 @@
 {% block content %}
 {% spaceless %}
 <div class="text-break m-4">
-    <p><a href="{% url 'documents' %}">{% trans "Documents" %}</a> > {{ document.title }}</p>
+    <nav aria-label="breadcrumb">
```

Comment:
Small gotcha that we haven't discussed much: i18n.

The aria label will be read to the user as natural language text, so it should be marked for translation, i.e.
```
    <nav aria-label="{% trans 'Breadcrumb' %}">
```
(untested)
I would capitalise, although it doesn't matter that much. Just make sure it is identical for the other similar places as well.

Reply:
Good point, thanks!